File: cpp/omegaup.com/Pivote-Reloaded.cxx
#include <iostream>

using namespace std;
int main(){
    int n, pivote;
    cin >> n;
    int v[n];
    for(int i = 0; i < n; i++){
        cin >> v[i];
    }
    cin >> pivote;
    for(int i = 0; i < n; i++){
        if(v[i] < pivote){
            cout << v[i] << " ";
        }     
    }
    cout<<endl;
    for(int i = 0; i < n; i++){
        if(v[i] > pivote){
            cout << v[i] << " ";
        }     
    }
    return 0;
}

File: Python/omegaup.com/Pivote-Reloaded.py
n = int(input())
v = list(map(int, input().split()))[:n]
pivote = int(input())

izquierda = []
derecha = []
for i in range(n):
    if v[i] < pivote:
        izquierda.append(v[i])
    
    if v[i] > pivote:
        derecha.append(v[i])

print(*izquierda)
print(*derecha)